tailwind.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        overlayColor:'#01010180',
```

Comment:
Looks Great! just make sure to add this 'overlayColor' to each `overlayClassName` and then we should be RFM!